clients/native/src/client/mod.rs

```diff
@@ -103,8 +102,13 @@ impl NymClient {
             topology_accessor,
         );
 
-        if self.config.get_base().get_use_extended_packet_size() {
-            stream.set_custom_packet_size(PacketSize::ExtendedPacket)
+        if let Some(size) = self.config.get_base().get_use_extended_packet_size() {
```

Comment:
I'd argue that if user provided any invalid value in their config we should blow up